syz-cluster/dashboard/handler.go

```diff
@@ -393,6 +393,9 @@ func errToStatus(f func(http.ResponseWriter, *http.Request) error) http.HandlerF
 		} else if errors.Is(err, errBadRequest) {
 			http.Error(w, err.Error(), http.StatusBadRequest)
 		} else if err != nil {
+			// TODO: if the error happened in the template, likely we've already printed
```

Comment:
You can reuse `newGzipResponseWriterCloser` to address it.
It buffers the output allowing you to react to the error.